docs/index.md

```diff
@@ -1,8 +1,12 @@
 # Welcome to the River Basin Model (RBM) website!
 
-On this website you will find documentation both about the model and how to run the model.  (talk more about it)
+On this website you will find documentation both about the model and how to run the model.
 
-# Documentation:
+RBM is a one-dimensional stream-temperature model that solves the time-dependent equation for the transfer of thermal energy across the air-water interface ([Yearsley, 2009, 2012](Documentation/References.md)). The model uses a semi-Lagrangian, particle tracking numerical scheme that is highly scalable in space and time. The model software was written initially for purposes of developing the Total Maximum Daily Load (TMDL) for water temperature in the Columbia River system ([Yearsley, 2003](Documentation/References.md)). The model had been modified to make use of the output from the large-scale hydrologic model VIC and regional scale model DHSVM.
```

Comment:
I just double checked, and the "Yearsley, 2003" is actually 2002 - it's listed as 2002 in your references, but I think it's actually 2002.  I'll send you the paper.